indico/htdocs/js/custom/dropzone.js

```diff
@@ -94,6 +95,14 @@
                         if (options.submit_form) {
                             $form.submit();
                         }
+                        if (options.handleFlashes) {
+                            handleFlashes(response, true, $flashArea);
+                        }
+
+                        $dz.data('value', response.content);
+                        $form.find('.change-trigger').val($dz.data('value') ? 1 : 0);
+                        $form.trigger('indico:fieldsSaved', response);
+                        $form.trigger('change');
```

Comment:
This is needed in order to ensure `data-disabled-until-change` works. There has to be a change event, and since the dropzone doesn't do it natively, we have to trigger it.